doc/sphinxext/cited_mne.py

```diff
@@ -37,12 +44,12 @@
       'Gecko/20100913 Firefox/3.6.9')
 
 # ##### Templates for citations #####
-html = (u""".. _cited
+html = (u""".. _cited:
 
 Publications by users
 =====================
 
-Papers citing MNE as extracted from Google Scholar (on %s).
+Papers citing MNE (%d) as extracted from Google Scholar (on %s).
```

Comment:
There are 66 MB in `mne/io/tests/data`, and the big files [have been around for 3 years](https://github.com/mne-tools/mne-python/tree/master/mne/io/tests/data)